utils/io/content/contentwriter.go

```diff
@@ -55,11 +59,25 @@ func NewContentWriter(arrayKey string, isCompleteFile, useStdout bool) (*Content
 	return &self, nil
 }
 
+func NewEmptyContentWriter(arrayKey string, isCompleteFile, useStdout bool) (*ContentWriter, error) {
```

Comment:
Let's add a description for this function, explaining the difference between this function and the *NewContentWriter* function.

Reply:
Removed